__tests__/app.test.js

```diff
@@ -38,6 +38,28 @@ describe("GET Endpoints", () => {
     });
   });
 
+  describe("/api/articles/ (05)", () => {
+    //returns an article slected by id
+    it("200: Returns object with a single article, selected by id", () => {
+      return request(app)
+        .get("/api/articles/2")
+        .expect(200)
+        .then((response) => {
+          const { article } = response.body;
+          expect(article).toBeInstanceOf(Object);
+          expect(article).toMatchObject({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
```

Comment:
I would expect that if i requested article 2, then the article_id should probably be exactly 2